src/classes/ADDR_Addresses_TEST.cls

```diff
@@ -61,7 +61,17 @@ private with sharing class ADDR_Addresses_TEST {
     /*********************************************************************************************************
     * @description The list of created test Addresses.
     */
-    private static list<Address__c> listAddrT;           
+    private static list<Address__c> listAddrT;
+    
+    private static ID orgRecTypeID;
+    private static ID householdRecTypeID;
+    private static ID adminRecTypeID;
+    
+    public static void recTypesSetup() {
```

Comment:
We don't include any Account record types in the package, so we just query to see if they are there. If the record types aren't there the tests will fail, which is why at some point I proposed included those record types that are used in tests to be included in the package. But we decided not to do that.